ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleExtrasBuilder.java

```diff
@@ -76,13 +76,21 @@ public Bundle build() {
   }
 
   public static AdConfig adConfigWithNetworkExtras(Bundle networkExtras) {
+    return adConfigWithNetworkExtras(networkExtras, null);
```

Comment:
Comment not related to this code part, but suggest simply fix this following 6.5.2.0 tag:
https://github.com/Vungle/googleads-mobile-android-mediation/blob/61f01ede097f364c592c2aa98bb8788ce6ad8fd9/ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleExtrasBuilder.java#L69-L82
https://github.com/Vungle/googleads-mobile-android-mediation/blob/61f01ede097f364c592c2aa98bb8788ce6ad8fd9/ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java#L249-L252

Reply:
I like @HuFangshuai 's suggestion to simply revert back to how it was earlier, was much simpler and clean logic